oqupy/bath_correlations.py

```diff
@@ -390,6 +390,21 @@ class CustomSD(BaseCorrelations):
         Whether or not to use a sine/cosine weighted version of the
```

Comment:
Suggest small revision to mention scipy,

"Whether to use an alternative integration scheme leveraging sine/cosine weighted versions of scipy.integrate.quad to handle rapid oscillations. This may improve numerical accuracy and stability, especially for long memory times.
[Blank line]
The integral...
"